--- ecp-cookie-init	2020-12-03 21:36:44.000000000 +0000
+++ ecp-cookie-init	2020-12-10 02:53:31.480000000 +0000
@@ -57,12 +57,12 @@
 # klist is required to check for valid kerberos ticket
 # tempfile or mktemp is required for safe temporary files
 
-type -P /usr/bin/curl >&/dev/null || { echo "This script requires curl. Aborting." >&2; exit 1; }
-type -P /usr/bin/xsltproc >&/dev/null || { echo "This script requires xsltproc. Aborting." >&2; exit 1;}
+type -P /opt/anaconda1anaconda2anaconda3/bin/curl >&/dev/null || { echo "This script requires curl. Aborting." >&2; exit 1; }
+type -P /opt/anaconda1anaconda2anaconda3/bin/xsltproc >&/dev/null || { echo "This script requires xsltproc. Aborting." >&2; exit 1;}
 
-curl_command=/usr/bin/curl
-xsltproc_command=/usr/bin/xsltproc
-klist_command=/usr/bin/klist
+curl_command=/opt/anaconda1anaconda2anaconda3/bin/curl
+xsltproc_command=/opt/anaconda1anaconda2anaconda3/bin/xsltproc
+klist_command=/opt/anaconda1anaconda2anaconda3/bin/klist
 
 VERSION="1.3.7"
 
@@ -139,8 +139,8 @@
 	exit 1
     fi
 
-    cookie_file_mode=$(/usr/bin/python -c "import os, stat; print(os.stat(\"${cookie_file}\")[stat.ST_MODE])")
-    cookie_file_uid=$(/usr/bin/python -c "import os, stat; print(os.stat(\"${cookie_file}\")[stat.ST_UID])")
+    cookie_file_mode=$(/opt/anaconda1anaconda2anaconda3/bin/python -c "import os, stat; print(os.stat(\"${cookie_file}\")[stat.ST_MODE])")
+    cookie_file_uid=$(/opt/anaconda1anaconda2anaconda3/bin/python -c "import os, stat; print(os.stat(\"${cookie_file}\")[stat.ST_UID])")
 
     if [ $cookie_file_uid -ne $(id -u) ]; then
 	echo "ERROR: cookie file $cookie_file not owned by $(whoami)"
